package.json

```diff
@@ -118,13 +118,15 @@
   },
   "main": "./dist/cjs/index.cjs",
   "exports": {
-    "require": {
-      ".": "./dist/cjs/index.cjs",
-      "types": "./dist/index.d.cts"
-    },
-    "import": {
-      ".": "./dist/esm/index.mjs",
-      "types": "./dist/index.d.mts"
+    ".": {
+      "require": {
+        "default": "./dist/cjs/index.cjs",
+        "types": "./dist/index.d.cts"
+      },
+      "import": {
+        "default": "./dist/esm/index.mjs",
+        "types": "./dist/index.d.mts"
+      }
```

Comment:
I don't know why this change was necessary, but windows is happy with this, whereas it was not happy with the previous code.